src/compiler/parser/index.js

```diff
@@ -306,6 +333,10 @@ export function parse (
         element.end = end
       }
       closeElement(element)
+      // check slot constraints
+      if (!stack.length && element === root) {
+        checkSlotConstraints(element, {})
```

Comment:
maybe the function could do the check and the warning could be executed here.
It would be better if the warning could also show some information about the Component where the parsing error was found.
Maybe using `warnOnce` is also a good idea

Reply:
OK, I resolved it.